package.json

```diff
@@ -27,6 +27,7 @@
     "eslint-plugin-filenames": "^0.1.1",
     "eslint-plugin-mocha": "^0.2.2",
     "eslint-plugin-one-variable-per-var": "0.0.3",
+    "hmpo-model": "^0.4.0",
```

Comment:
Yeah, just to get the correct event behaviour for a mock `sessionModel` - change:key events